src/gateway/request.js

```diff
@@ -51,10 +51,6 @@ export const request = (
      * Then we capture the abort function so we can reattach it later.
      */
     const traceSession = trace(`request`, url, requestLogger);
-    // It would be nice if the metadata of the logger was automatically
-    // added as a trace label, but for child loggers, that's hard since
-    // the child metadata isn't currently accessible off the logger;
-    // the defaultMetadata prop is only the root logger's metadata.
```

Comment:
I got things the wrong way round about how stuff works - and this comment just wasn't relevant anymore.